app.js

```diff
@@ -1,14 +1,13 @@
 const express = require('express');
 const path = require('path');
 const routeLogin = require('./routes/login');
-
 var database = require('./db/database');
```

Comment:
always split the require lines from actual code by adding a empty line between them